docs/Tutorials/DomainCreation.md

```diff
@@ -184,28 +184,28 @@ are:
 ```
 
 //Folding highest cube downward:
-{{53,54,69,70},{53,54,40,41}},
+{{53,54,69,70},{53,54,40,50}},
```

Comment:
Thanks for checking this, I didn't realize how many mistakes/typos there were...

I think `{{53,54,69,70},{53,54,40,50}}` should be `{{53,54,69,70},{53,54,49,50}}`?

Reply:
It should be, you're right.